src/components/lessonContents/QuestResults.tsx

```
@@ -1,67 +1,76 @@
-                    chestLevel={1}
-                />
+
+                    return (
+                        <DailyQuestCard
+                            type={dailyQuestDefinition.requirementType}
+                            title={dailyQuestDefinition.title}
+                            progress={dailyQuestProgress}
+                            currentValue={userDailyQuestProgress?.progress}
+                            targetValue={dailyQuestDefinition.requirement}
+                            chestLevel={index + 1}
+                        />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)